com.unity.visualeffectgraph/Editor/Inspector/VFXSlotContainerEditor.cs

```diff
@@ -279,8 +279,8 @@ public class Contents
         public static GUIContent name = EditorGUIUtility.TrTextContent("Name");
         public static GUIContent type = EditorGUIUtility.TrTextContent("Type");
         public static GUIContent mode = EditorGUIUtility.TrTextContent("Mode");
-        public static GUIContent gizmoLocalWarning = EditorGUIUtility.TrIconContent(EditorGUIUtility.Load(EditorResources.iconsPath + "console.warnicon.sml.png") as Texture2D, "Local values require a target GameObject to display");
-        public static GUIContent gizmoIndeterminateWarning = EditorGUIUtility.TrIconContent(EditorGUIUtility.Load(EditorResources.iconsPath + "console.warnicon.sml.png") as Texture2D, "The gizmo value is indeterminate.");
+        public static GUIContent gizmoLocalWarning = EditorGUIUtility.TrIconContent(EditorGUIUtility.LoadIconForSkin(EditorResources.iconsPath + "console.warnicon.sml.png", EditorGUIUtility.skinIndex), "Local values require a target GameObject to display");
```

Comment:
I see there exists a method LoadIcon(string name) that does the same thing, but without requiring a second argument. Did you miss it or is there another reason for setting the skin explicitly ?

Reply:
I missed it, good catch ! Thank you :)